<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Score</class>
 <widget class="QWidget" name="Score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="jumpToMainWindowP">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回首页</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>191</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="scoreL">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分数</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>210</y>
     <width>541</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>建议：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="suggestionL">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>建议内容</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
